akka-stream/src/main/mima-filters/2.6.11.backwards.excludes/26622-stagelogic-internal-api.excludes

```diff
@@ -0,0 +1,2 @@
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.stage.GraphStageLogic.getEagerStageActor")
```

Comment:
this should now be renamed to 2.6.12

Reply:
rebased to latest master and moved file to 2.6.12